vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -265,6 +265,10 @@ Error.VSQL_too_old_title=Incompatible VSQL Version
 Error.VSQL_too_old_heading=Incompatible VSQL Version
 Error.VSQL_too_old_message=<html>The verison of VSQL you are trying to load is incompatible with VASSAL %1$s. Please check <a href="https://vassalengine.org/wiki/Module:Squad_Leader">vassalengine.org</a> for a newer version of VSQL, or load this version of VSQL with VASSAL 3.2.17.</html>
 
+Error.POG_too_old_title=Please update Paths of Glory module
+Error.POG_too_old_heading=Please update Paths of Glory module
+Error.POG_too_old_message=<html>The verison of Paths of Glory you are trying to load is incompatible with VASSAL %1$s. Please check <a href="https://vassalengine.org/wiki/Module:Paths_of_Glory">vassalengine.org</a> for a newer version of Paths of Glory.</html>
```

Comment:
verison

Reply:
Lol on verison -- apparently cut-and-pasting from the VASL stuff was a mistake haha. Fixed VASL and VSQL versions too.

I certainly agree if a more generic way of setting lowest-loadable-version for a module was available that it would be a good idea. My throwaway idea would be something on the wiki page.